server/README-ECLIPSE.md

```diff
@@ -0,0 +1,40 @@
+Eclipse Installation
+--------------------
+
+The installation of the various plug-ins we use for development,
+code quality, etc., is straightforward, but is nowhere near as automated
+as we would like. To install an Eclipse distribution with all the required
+tools and get the project ready for development, do the following:
+
+  1. Download the Eclipse Oxygen installer for your platform from
+  http://www.eclipse.org/oxygen/.
+  2. Run the installer and install the "Eclipse IDE for Java Developers"
+  (the first option on the list of distributions to install).
```

Comment:
@nealmcb I'll open an issue for this so we capture the concern, which is valid, and also keep things movin' along!